src/main/java/generator/BaseScenarioGenerator.java

```diff
@@ -99,6 +99,8 @@ private void addVehicle(Model model) {
         vehicle1.addAccelerationY(0F);
         vehicle1.addAccelerationX(0F);
         vehicle1.addValueInDollars(1000000F);
+        vehicle1.addIs_on_lane(lane);
+        lane.addLane_has_vehicle(vehicle1);
 
```

Comment:
Tak ustawiamy właściwości dzięki czemu możemy odczytać w ScenarioRreaderze scenariusz z ontologii i załadować do modelu

Reply:
property is_on_lane było w ontologii juz wczesniej, nie?

Reply:
O ile dobrze pamiętam to było, właśnie dodałem on_lane kiedyś bo tego is_on_lane nie zauważyłem także pasowałoby to on_lane_usunąć 
![image](https://user-images.githubusercontent.com/47008811/100910753-a0c36780-34ce-11eb-9800-9dbe8f24d550.png)

Reply:
my w kazdym razie is_on_lane uzywamy